sources/models/shared-defaults/kubernetes-aws.toml

```diff
@@ -13,8 +13,15 @@ provider-id.setting-generator = "pluto provider-id"
 affected-services = ["kubernetes"]
 
 [metadata.settings.kubernetes.pod-infra-container-image]
-setting-generator = "schnauzer-v2 render --requires 'aws@v1' --requires 'kubernetes@v1(helpers=[pause-prefix])' --template '{{ pause-prefix settings.aws.region }}/eks/pause:3.1-eksbuild.1'"
-affected-services = ["kubernetes", "containerd"]
+affected-services = ["pod-infra-container-image"]
+
+[services.pod-infra-container-image]
+configuration-files = ["pod-infra-container-image-log-message"]
+restart-commands = ["systemctl restart deprecation-warning@pod-infra-container-image.timer"]
+
+[configuration-files.pod-infra-container-image-log-message]
+path = "/etc/deprecated-settings/pod-infra-container-image"
+template-path = "/usr/share/templates/pod-infra-container-image"
```

Comment:
Here we're creating new "prefixes" in the datastore - which roughly correspond to maps with some number of (possibly nested) key / value pairs, or (equivalently) directories with files (keys), file contents (values), and subdirectories (nested maps).

These end up as directories in the data store:
```
/var/lib/bottlerocket/datastore/current/live/services/pod-infra-container-image/...
/var/lib/bottlerocket/datastore/current/live/configuration-files/pod-infra-container-image-log-message/...
```

Those directories will be auto-populated by a program that uses this file on new launches and upgrades, but need to be removed on downgrade.

`AddPrefixesMigration` (as shown [here](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/api/migration/migrations/v1.15.0/deprecate-log4j-hotpatch-enabled/src/main.rs)) is the migration helper for this. It's roughly like an `rm -rf` on those directories.

```
    migrate(AddPrefixesMigration(vec![
        "services.pod-infra-container-image",
        "configuration-files.pod-infra-container-image-log-message",
    ]))
```